From 23482f788665df83edc8b542698f45fed45a2711 Mon Sep 17 00:00:00 2001
From: Carlos O'Donell <carlos@redhat.com>
Date: Fri, 2 Oct 2020 09:23:35 -0400
Subject: [PATCH] Set version.h RELEASE to "stable" (Bug 26700)

The RELEASE macro was accidentaly set to "release" instead of
the expected "stable" by the release manager.  This is a mistake
that leads to the build using "-g -O1" instead of "-g -O2" if
configure was executed with "CFLAGS=" (CFLAGS set but empty).
---
 version.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/version.h
+++ b/version.h
@@ -1,4 +1,4 @@
 /* This file just defines the current version number of libc.  */
 
-#define RELEASE "release"
+#define RELEASE "stable"
 #define VERSION "2.33"
